integration-tests/src/test/java/tachyon/master/lineage/LineageMasterIntegrationTest.java

```diff
@@ -142,7 +142,7 @@ public void lineageCompleteAndAsyncPersistTest() throws Exception {
           fileInfo.getPersistenceState());
 
       // sleep and wait for worker to persist the file
-      CommonUtils.sleepMs(5);
+      CommonUtils.sleepMs(200);
```

Comment:
I agree with you. The hard part is to be able to control the completion of the thread. Added a todo.